README.md

```diff
@@ -274,6 +274,38 @@ package "runtime-lib" ( ... )
 You can check that this META works for all the 4 methods described
 above.
 
+### Building rewriter that you are currently developing
+When developing a new rewriter you are very likely to prepare a few
```

Comment:
Shouldn't this section come after `Building a custom driver using ocamlfind`? For instance it's not clear to me what `the compilation line above` refers to.